test/e2e/internal/utils/env.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"os"
+)
+
+// TempUserDir creates a tmp directory.
+//
+// Calling cleaner will remove the directory recursively.
+func TempUserDir() (userDir string, cleaner func() error, err error) {
+	// generate random temp dir for notation
+	userDir, err = os.MkdirTemp("", "e2e-")
```

Comment:
Why not using https://pkg.go.dev/github.com/onsi/ginkgo/v2@v2.1.6?utm_source=gopls#GinkgoTInterface.TempDir?

Reply:
Updated.